metadata-ingestion/src/datahub/ingestion/source/sql/mssql/source.py

```diff
@@ -237,6 +270,343 @@ def create(cls, config_dict: Dict, ctx: PipelineContext) -> "SQLServerSource":
         config = SQLServerConfig.parse_obj(config_dict)
         return cls(config, ctx)
 
+    def get_ucs(self, inspector: Inspector, key: str) -> List[UpstreamClass]:
```

Comment:
what is ucs? can we expand out the acronym

Reply:
This is acronym of UpstreamClass -> **U**pstream**C**las**s** -> ucs. Can rename if it is confusing)

Reply:
Let's just call it `get_upstreams`